cypress/e2e/osd/OSDClusterCreation.js

```diff
@@ -17,9 +17,9 @@ describe('OSD cluster tests', { tags: ['ci'] }, () => {
     it('navigates to create OSD cluster', () => {
       cy.getByTestId('create_cluster_btn').click();
       CreateClusterPage.isCreateClusterPage();
-      cy.get('a[data-testid="osd-create-cluster-button"]', {
-        timeout: 15000,
-      }).click();
+      cy.getByTestId('osd-create-cluster-button', { timeout: 50000 })
+        .should('not.have.class', 'pf-m-aria-disabled')
```

Comment:
I think it's better to check directly the `aria-disabled` attribute instead of the pf class. In the end the result is the same but they are meant for different purposes. In this case it feels wrong to mix them.

Reply:
If it is the only possible method to select the component, that is fine, but as the guidelines suggest, we should mention that somewhere (adding a comment about it on the PR description / on the PR's code as a comment would be fine for me 😄)

In case we have a different selector we could use, that would be the approach to go for in my opinion to avoid adding attributes we could avoid to the components we use

Reply:
If you are referring the unittest guidelines. At the moment, we’re not following this approach in our Cypress test definitions. 🙂

Reply:
The use of `.should('not.have.attr', 'aria-disabled')` isn't working reliably in our scenario.
As an alternative, we can invoke the attribute directly, check for its absence, and then perform the click. 
For example:
```
cy.getByTestId('osd-create-cluster-button')
  .invoke('attr', 'aria-disabled')
  .should('not.exist')
  .then(() => {
    cy.getByTestId('osd-create-cluster-button').click();
  });
 ```
However, I found that relying on the class definition makes things simpler

Reply:
IRT:

> we can invoke the attribute directly, check for its absence, and then perform the click

yes - that's what i would go with actually.   i've seen similar issues after the move to PF 6 (where aria-disabled is simply omitted) in another area

Reply:
> The use of .should('not.have.attr', 'aria-disabled') isn't working reliably in our scenario.

That's probably because you have to check if `aria-disabled` is undefined OR false to cover all scenarios.

> However, I found that relying on the class definition makes things simpler

True but that shouldn't be the only criteria to consider if there are alternatives. 

> oh BTW, does simply .should('be.disabled') not work for this case?

That doesn't work on `<a>` links because you can't disable them. You can trick the browser disabling pointer events but they don't support a `disabled` attribute.

Reply:
> That doesn't work on <a> links because you can't disable them

right.. the problem makes more sense now